admin-dev/themes/new-theme/scss/pages/orders/_orders_main.scss

```diff
@@ -99,13 +99,12 @@
   margin-top: 8px;
 }
 
-$order-block-background-color: #f8f8f8 !default;
```

Comment:
So if a theme inherit, it won't be able to use this variable ?

Reply:
yes right

Reply:
So could you put it back or define it as a BC Break ?